### PR TITLE
Update ProductVariantName components "name" form field into "variantName"

### LINKED ISSUE #2831: The variant name changes when entering value in the variant attributes
### The variant name changes when entering value in the variant attributes fields on the variant detail page.
The variant name should not change when entering the attribute value.
### Steps to reproduce the problem

1. The user logs into the Saleor.
2. User clicks "Products" button from the menu.
3. User clicks a product on the products page.
4. User clicks "Edit variant " button.
5. User changes variant attributes in the variant detail page. 

### Expected Result

Variant name should not change

### Actual Result

It was seen that the variant name has changed. 

### Screenshots

Screen Shot 2022-12-09 at 14 32 28

### System information

Dashboard version:

v3.8.7

Core version:

v3.8.17

Operating system:

- [ ] MacOS
- [ ] Windows

Browser:

- [ ] Chrome

### PR DESCRIPTION
**Pull request by ilkaykisayol**
I want to merge this change because, it fixes variant name changes when entering value in the variant attributes

 
Fixes #2831 
**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
MARKETPLACE_URL=https://apps.saleor.io
SALEOR_APPS_ENDPOINT=https://apps.saleor.io/api/saleor-apps

### Do you want to run more stable tests?
To run all tests, just select the stable checkbox. To speed up tests, increase the number of containers. Tests will be re-run only when the "run e2e" label is added.

1. [ ] stable
2. [ ] giftCard
3. [ ] category
4. [ ] collection
5. [ ] attribute
6. [ ] productType
7. [ ] shipping
8. [ ] customer
9. [ ] permissions
10. [ ] menuNavigation
11. [ ] pages
12. [ ] sales
13. [ ] vouchers
14. [ ] homePage
15. [ ] login
16. [ ] orders
17. [ ] products
18. [ ] app
19. [ ] plugins
20. [ ] translations
21. [ ] navigation
22. [ ] variants
23. [ ] payments

CONTAINERS=1

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-2831 PluginsManager should call action only on active plugin](https://app.clickup.com/t/2549495/SALEOR-2831)

**Comment by ogunheper** [comment]
Hi @patrys ,

Can someone take a look at this PR please? We need this in production.

Best regards,
Ogün

**Review by poulch — DISMISSED** [review]

File: src/products/components/ProductStocks/ProductStocks.tsx [hunk 1]
```diff
@@ -131,6 +131,16 @@ const ProductStocks: React.FC<ProductStocksProps> = ({
...
   };
 
+  const handleChange = e => {
+    onFormDataChange(e);
+    onFormDataChange({
+      target: {
+        name: "variantName",
+        value: e.target.value,
+      },
+    });
```
Comment (poulch) [inline src/products/components/ProductStocks/ProductStocks.tsx:141]: Why do you call `onFormDataChange ` twice ?

Comment (ilkaykisayol) [reply src/products/components/ProductStocks/ProductStocks.tsx:141]: Hi @poulch, we are expecting variant name field changes when sku field is updated. we would like to keep this behavior.

File: src/products/components/ProductStocks/ProductStocks.tsx [hunk 2]
```diff
@@ -131,6 +131,16 @@ const ProductStocks: React.FC<ProductStocksProps> = ({
     }
   };
 
+  const handleChange = e => {
```
Comment (poulch) [inline src/products/components/ProductStocks/ProductStocks.tsx:134]:
```suggestion
  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```

**Review by poulch — APPROVED** [review]

**Review by poulch — APPROVED** [review]

**Review by Droniu — APPROVED** [review]

**Review by krzysztofzuraw — APPROVED** [review]

**Comment by ogunheper** [comment]
Hi @poulch ,

Can we merge this if all is well? Afterwards we plan to create another PR towards 3.9 branch for the 3.9 port.

Best regards,
Ogün

**Comment by karolm-saleor** [comment]
√ QA Passed

**Review by karolm-saleor — APPROVED** [review]

**Comment by poulch** [comment]
FYI @ogunheper your changes have been deployed in 3.10.0 release.
https://github.com/saleor/saleor-dashboard/releases/tag/3.10.0

**Pull request merged**